doc/next/_sources/reference/delete-campaign-linkedads.rst.txt

```diff
@@ -0,0 +1,39 @@
+.. index:: DELETE /campaign/{id}/linkedAds
+
+
+TODO: WE DO NOT YET HAVE AN IMPLEMENTATION FOR A SINGLE REMOVAL, ONLY LIST!
+
+
+.. _delete_campaign_linkedads:
+
+DELETE /campaign/{id}/linkedAds
+===============================
+
+It is possible to remove a :ref:`single <delete-single-linked-ad>` linkedAd or :ref:`multiple <delete-multiple-linked-ads>` linkedAds from a campaign in one go. Removing the linkedAd from a campaign does not remove the ad from your inventory; it only takes the ad offline for that particular campaign.
```

Comment:
Should we annotate this creates orphaned ads, that may or may not show in the console or even anywhere? Given that their UI's probally also dont show orphans

Reply:
wait wait, this is not released in the first stage. I didn't remove the doc but removed the references (or do you still see them somewhere?)

Reply:
nothing about linkedAds in the first phase, there should be part of the doc explicitely saying we're handling the ownership of ads in a campaign for now

Reply:
Yhea, I was just commenting ahead of this being released :D

Reply:
yeah I'm kinda hoping that by then we'll rework the architecture to avoid having these orphans :S